cli.js

```diff
@@ -11,7 +11,8 @@ const localCLI = resolveCwd('ava/cli');
 // Use `path.relative()` to detect local AVA installation,
 // because __filename's case is inconsistent on Windows
 // see https://github.com/nodejs/node/issues/6624
-if (localCLI && path.relative(localCLI, __filename) !== '') {
+process.env.AVA_LOCAL_CLI = process.env.AVA_LOCAL_CLI || (localCLI && path.relative(localCLI, __filename) !== '');
```

Comment:
Oh I didn't know that. When I was debugging it, it was a automatically converted to a string.
Its changed now.